_pages/dev/features/product-multi-dimensional.md

```diff
@@ -0,0 +1,197 @@
+
+```ts
+    provideDefaultConfig(<CmsConfig>{
+    cmsComponents: {
+        ProductMultiDimensionalSelectorComponent: {
+            component: ProductMultiDimensionalSelectorComponent,
+            guards: [ProductMultiDimensionalSelectorGuard],
+        },
+    },
+}),
+```
+
+Multi-dimensional products need to have the `ProductMultiDimensionalSelectorComponent` CMS component present in the `/page` response, and they need to be rendered on the page.
+
+### Adding a Multi-Dimensional Selector Using ImpEx
+
+To add a multi-dimensional selector using ImpEx, you create a `ProductMultiDimensionalSelector` CMS flex component, and then add it to a page slot, such as the `ProductSummarySlot`.
+
+**Note:** The `ProductSummarySlot` is the recommended location for adding your `ProductMultiDimensionalSelector`, but you can add the `ProductMultiDimensionalSelector` to any page slot on the product details page (PDP) that you wish.
+
+Variant elements for multi-dimensional products appear in locations such as the following:
+
+- product details pages
+- cart items (for example, see the `CartItemComponent`)
+- account pages, such as the order page, the order history, wish list, cancellations and returns, and so on.
+
+**Note:** The `$contentCV` variable that is used in the following ImpEx examples, and which stores information about the content catalog, is defined as follows:
+
+```text
+$contentCatalog=powertools-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+```
+
+1. Create a `ProductMultiDimensionalSelector` CMS flex component with the following ImpEx:
+
+   ```text
+   INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType;&componentRef
+   ;;ProductMultiDimensionalSelector;Product Multi-Dimensional Selector Component;ProductMultiDimensionalSelectorComponent;ProductMultiDimensionalSelector
+   ```
+
+2. Add the newly created CMS flex component to the `ProductSummarySlot` with the following ImpEx:
+
+```text
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;cmsComponents(uid, $contentCV)
+;;ProductSummarySlot;Summary for product details;ProductImagesComponent,ProductIntroComponent,QualtricsEmbeddedFeedbackComponent,ProductSummaryComponent,ProductMultiDimensionalSelector,AddToCart,ConfigureProductComponent,AddToWishListComponent,StockNotificationComponent
+```
+
+## Images
+
+Variant categories can use images for selecting the categories by setting the `hasImage` attribute to true. This can be configured in Backoffice, or through ImpEx. The `hasImage` attribute acts as a toggle that informs the storefront to display images for variant options. However, images are only shown if each variant option within the category has an image in the correct format. The default image format for multi-dimensional products is `STYLE_SWATCH`, which is 30 W x 30 H.
+
```

Comment:
"This can be configured in Backoffice, or through ImpEx."

Are we able to, and would we want to provide an example of how to do this using ImpEx? 🙂

Reply:
we can provide an example, but I'm not sure if we should. Here is whole document about the categories https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/7e47d40a176d48ba914b50957d003804/8ae0f0ac86691014b4e0e53c678d8b44.html#creating-variantvaluecategorys-using-impex

```java
				# Insert Variant Category
				INSERT_UPDATE VariantCategory;code[unique=true];name;$supercategories;**hasImage**;$catalogVersion;allowedPrincipals(uid)[default='customergroup']
				;B2B_Color;Color;;true
				;B2B_Fit;Fit;B2B_Color;false
				;B2B_Size;Size;B2B_Fit;false
```

Reply:
Perfect! I added a link to this documentation. Thank you! 🙂